pkg/rke2/rke2.go

```diff
@@ -316,13 +317,11 @@ func checkForRunningContainers(ctx context.Context, tmpAddress string, disabledI
 			continue
 		}
 		var gc bool
-		for item, disabled := range disabledItems {
-			if disabled {
-				if isContainerRunning(ctx, item, resp) {
-					logrus.Infof("Waiting for deletion of %s static pod", item)
-					gc = true
-					break
-				}
+		for item, _ := range disabledItems {
```

Comment:
@galal-hussein I don't see where we're adding anything to the disabledItems list - if we're shutting down everything shouldn't we be populating this with all the basenames of manifests in the static pods dir?

Reply:
The disabled item list is already populated with all names, its actually a map of string booleans, right now we just check all of them and delete all manifests